index.js

```diff
@@ -223,6 +223,8 @@ function buildFetch(behavior) {
             return handleError(url, cacheKey, err.response, err.response.body, resolvedCallback);
           }
         } else if (err instanceof got.TimeoutError) {
+          const { timeout: timeoutOptions } = options ?? {};
```

Comment:
log the got timings object, the timeout options are not interesting (you set them yourself, but what phase that timed out is interesting; i.e. what the got timings object will tell you)